### DISCUSSION TITLE
Sorted results

### DISCUSSION THREAD

**Discussion by MariaMozgunova** [openpotato/openplzapi.website#7] [Q&A] [answered]
Can I assume the returned results are alphabetically sorted when querying data in JSON format?
It looks like the results are sorted by the `name` field, but I could not find a mention of this in the documentation.
Specifically, I am querying the `openplzapi.org/de/Streets?name={streetName}&postalCode={postalCode}&locality={localityName}` endpoint.

**Comment by fstueber** [comment, marked as answer]
Yes, its ordered by street name, then postal code, then locality. Will add this to the docs. Thanks for the feedback
